pkg/collector/corechecks/oracle-dbm/statements.go

```diff
@@ -194,6 +195,33 @@ type MetricsPayload struct {
 	OracleVersion string      `json:"oracle_version,omitempty"`
 }
 
+type FQTDBMetadata struct {
+	Tables   []string `json:"dd_tables"`
+	Commands []string `json:"dd_commands"`
+}
+
+type FQTDB struct {
+	Instance       string        `json:"instance"`
+	QuerySignature string        `json:"query_signature"`
+	Statement      string        `json:"statement"`
+	FQTDBMetadata  FQTDBMetadata `json:"metadata"`
+}
+
+type FQTDBOracle struct {
+	CDBName string `json:"cdb_name,omitempty"`
+}
+
+type FQTPayload struct {
+	Timestamp    float64     `json:"timestamp,omitempty"`
+	Host         string      `json:"host,omitempty"` // Host is the database hostname, not the agent hostname
+	AgentVersion string      `json:"ddagentversion,omitempty"`
+	Source       string      `json:"ddsource"`
+	Tags         []string    `json:"tags,omitempty"`
+	DBMType      string      `json:"dbm_type"`
+	FQTDB        FQTDB       `json:"db"`
```

Comment:
for the future: FQT events should follow the same exact structure [as explain events](https://github.com/DataDog/dd-go/blob/prod/database-monitoring/apps/dbm-events-processor/decoder/payloads.go#L302-L317) (minus just a few fields, of course). It probably makes sense to define a shared generic struct that we can use between this and the thread that is collecting execution plans

Reply:
Created a [Jira ticket](https://datadoghq.atlassian.net/browse/DBM-1595) for that.